source/store/test/AppCompatibilityContext.test.ts

```diff
@@ -0,0 +1,56 @@
+import {
+  AppCompatibilityVisitor,
+  APPLICATION_COMPATIBILITY_STATUS,
+} from "app/types/AppCompatibilityTypes";
+import { createAppCompatibilityContextValue } from "../AppCompatibilityContext";
+
+describe("AppCompatibilityContext", () => {
+  it("visit(visitor) calls correct visitor method", () => {
```

Comment:
A normally used patter to follow when writing jest tests is the ARRANGE, ACT and ASSERT syntax.

```js
it('a test', () => {
 // Arrange
 const expectedResult = "";
 const myFunctionCallback = "";
 ... other variables  

// Act
  const result = myFunction(...);
  
  // Assert
  expect(result).toEqual(expectedResult);
});

```

Notice the newlines between each block